backend/hardware/devicelist.py

```diff
@@ -52,11 +50,11 @@ def setupDevices(deviceDefinitions):
     return devices
 
 
-def validateConfiguration(deviceConfigs):
+def validateConfiguration(deviceConfigs: list[dict]):
     deviceDict = {}
     for device in deviceConfigs:
 
-        if deviceDict.get(device['id'], None) == None:
+        if deviceDict.get(device['id'], None) is None:
```

Comment:
The comparison with `==` will definitely work but it is my understanding that the 'is' check is more encouraged now for `None` or `True` and `False`